setup/dependencies.sh

```diff
@@ -0,0 +1,28 @@
+if [ -z "$SUDO_USER" ]; then
+    echo "This script is intended to be run as sudo. Exiting"
+    exit 1
+fi
+
+# TODO: detect and toggle between bash and zsh
+
+add-apt-repository -y ppa:deadsnakes/ppa
+apt update
+apt install -y python3.9 python3.9-dev python3-pip python3-venv
```

Comment:
Why are these needed in addition to installing Python 3.9 with pyenv?

Reply:
This is also a copy/paste from the HQ Developer setup. Are none of these needed?